mkosi/resources/man/mkosi.md

```diff
@@ -1003,10 +1003,12 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
 `KernelCommandLine=`, `--kernel-command-line=`
 :   Use the specified kernel command line when building images.
 
-    If the value of this setting contains the literals `root=PARTUUID`
-    or `mount.usr=PARTUUID`, these are replaced with the partition UUID
-    of the root or usr partition respectively. For example,
-    `root=PARTUUID` would be replaced with
+    If the root or usr partition are created with verity enabled,
+    `roothash=` or `usrhash=` respecitvely are automatically added to the
+    kernel command line. Otherwise, if the value of this setting
```

Comment:
```suggestion
    If the root or usr partition are created with verity enabled,
    `roothash=` or `usrhash=` respecitvely are automatically added to the
    kernel command line and `root=` or `usr=` should not be added.
    Otherwise, if the value of this setting
```